EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -1481,7 +1478,7 @@ void V2World::allocateFactories(const EU4::world& sourceWorld)
 
 		// modified manufactory weight follows diminishing returns curve y = x^(3/4)+log((x^2)/5+1)
 		int manuCount = sourceCountry->getManufactoryCount();
-		double manuWeight = pow(manuCount, 0.75) + log((manuCount * manuCount) / 5.0 + 1.0);
+		double manuWeight = pow(manuCount, 0.75) + log1p(((double)manuCount * (double)manuCount) / 5.0);

```

Comment:
Might just want to have manuCount as double (use a C++ style cast when getting the count).

Reply:
static recast.